lib/ops/opsservice/configure.go

```diff
@@ -1172,26 +1080,38 @@ func toObject(in interface{}) (map[string]interface{}, error) {
 
 func (s *site) teleportMasterConfigPackage(master remoteServer) (*loc.Locator, error) {
 	configPackage, err := loc.ParseLocator(
-		fmt.Sprintf("%v/%v:0.0.1-%v", s.siteRepoName(), constants.TeleportMasterConfigPackage,
-			PackageSuffix(master, s.domainName)))
+		fmt.Sprintf("%v/%v:0.0.%v-%v", s.siteRepoName(), constants.TeleportMasterConfigPackage,
+			time.Now().UTC().Unix(), PackageSuffix(master, s.domainName)))
```

Comment:
I thought about this actually. I can see a use-case where we might need to update a teleport config w/o updating the package version, for example to update a value of some field (idk, like a timeout or something). Although the way it's implemented right now, the code does not really support this use-case (it will not rotate config unless teleport package version changed).